librad/src/git/storage/pool.rs

```diff
@@ -39,12 +39,24 @@ impl Deref for PooledRef {
     }
 }
 
+impl DerefMut for PooledRef {
```

Comment:
ach, unrelated.

I was trying to prevent same-thread access to the `DashMap` by borrowing `self` mutably, but that didn't work out.